timeseries/src/autogluon/timeseries/models/autogluon_tabular/tabular_model.py

```diff
@@ -97,64 +98,40 @@ def __init__(
     def _get_features_dataframe(
         self,
         data: TimeSeriesDataFrame,
-        last_k_values: Optional[int] = None,
+        max_rows_per_item: int = 100_000,
```

Comment:
wondering any reason why we choose 100k? It is less than 1 month of min level data